iron-fit-behavior.html

```diff
@@ -485,14 +485,19 @@
     },
 
     __getNormalizedRect: function(target) {
-      if (target === document.documentElement || target === window) {
+      if (target === window) {
```

Comment:
Why did this change from `target === document.documentElement || target === window`?